src/store/user.ts

```diff
@@ -100,31 +122,33 @@ export class UserModel {
   }
 
   /**
-   * Return the observable user's properties.
+   * @return the observable user's properties.
    */
   getProperties(): BehaviorSubject<User> {
     return this.properties_;
   }
 
   /**
-   * Return the current user state.
+   * @return the current user state.
    */
   getState(): UserState {
     return this.state_;
   }
 
   /**
    * Set the current User's properties and state.
+   * @param properties
+   * @param state
    */
   setUser(properties: User, state: UserState) {
     this.checkUserProperties_(properties);
-    console.assert(state == null);
+    console.assert(state === null);
```

Comment:
Well, `==` was the aim. But I was surprised to no have an error.
Anyway. Merge it like that. I'll set that to `state !== undefined && state !== undefined` in my PR about the tests